plugins/outputs/warp10/warp10.go

```diff
@@ -0,0 +1,293 @@
+package warp10
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"regexp"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/internal/tls"
+	"github.com/influxdata/telegraf/plugins/outputs"
+)
+
+const (
+	defaultClientTimeout = 15 * time.Second
+)
+
+// Warp10 output plugin
+type Warp10 struct {
+	Prefix  string
+	WarpURL string
+	Token   string
+	Timeout internal.Duration `toml:"timeout"`
+	client  *http.Client
+	tls.ClientConfig
+}
+
+var sampleConfig = `
+  # prefix for metrics class Name
+  prefix = "Prefix"
```

Comment:
Based on the style of some built in metrics, and because you probably want to end this with a `.`, let's set this to `prefix = "telegraf."?